scrc/dataset_creation/dataset_creator.py

```diff
@@ -630,7 +645,7 @@ def create_sub_datasets(self, splits, split_type):
             lower, higher = boundaries[i] + 1, boundaries[i + 1]
             sub_dataset = sub_datasets_dict['input_length'][f'between({lower:04d},{higher:04d})'] = dict()
             for split_name, split_df in splits.items():
-                sub_dataset[split_name] = split_df[split_df.num_tokens_bert.between(lower, higher)]
+                sub_dataset[split_name] = split_df[split_df.facts_num_tokens_bert.between(lower, higher)]
```

Comment:
hmm, this probably cannot be hard-coded. What if the feature cols are the full_text or the considerations?

Reply:
Yeah, didn't think about that. I just always had an error because of the `num_tokens_bert` but the problem is probably anywhere else so I undid it.